Header header
geometry_msgs/PoseStamped target_pose
ControllerMode control_mode
ContactInfo[] contact_info
